server/index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const initializeDatabase = require('./database/init.js');
+
+const app = express();
+const router = require('./router');
+
+app.use(express.static('public'));
```

Comment:
@krise3 @Pfdiamond13 This relative path could cause problems. Does this serve up static assets correctly? Might just default to always using absolute paths (i.e., the Path module); much less likely to cause headaches.